src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -32,10 +36,34 @@ public class ChatConfig {
     public ChatSymbols chatSymbols = new ChatSymbols();
 
     @Expose
-    @ConfigOption(name = "Dungeon Filter", desc = "Hides pickup, reminder, buff, damage, ability, puzzle and end messages in Dungeons.")
-    @ConfigEditorBoolean
-    @FeatureToggle
-    public boolean dungeonMessages = true;
+    @ConfigOption(name = "Dungeon Filter", desc = "Hide specific message types in Dungeons.")
+    @ConfigEditorDraggableList()
```

Comment:
You can remove the () at the end

Reply:
removed in [Commit](https://github.com/hannibal002/SkyHanni/pull/884/commits/09dd5658c28d09b221a5c88ccdc4e12a6fc06111) ty